### PR TITLE
Renaming examples to system setups

### PR DESCRIPTION
**Pull request by k-rieck**

### CODE REVIEW

File: hisim/utils.py [hunk 1]
```diff
@@ -49,11 +49,11 @@ def get_input_directory() -> str:
     ),
     "cfg": os.path.join(hisim_abs_path, "inputs", "cfg.json"),
     "utsp_results": os.path.join(hisim_results, "Results"),
-    "utsp_example_results": os.path.join(
+    "utsp_system setup_results": os.path.join(
```
Comment (k-knosala) [inline hisim/utils.py:52]: This needs an underscore.

File: hisim/utils.py [hunk 2]
```diff
@@ -49,11 +49,11 @@ def get_input_directory() -> str:
...
         hisim_inputs, "LPGResults_for_tests", "Results"
     ),
     "utsp_reports": os.path.join(hisim_results, "Reports"),
-    "utsp_example_reports": os.path.join(
+    "utsp_system setup_reports": os.path.join(
```
Comment (k-knosala) [inline hisim/utils.py:56]: See above

**Comment by k-knosala** [comment]
This also adds `setup_function` to all system setups and removes the argument `function_in_module` in hisim main.

**Review by k-knosala — COMMENTED** [review]
Looks good to me apart from the two comments above.

File: hisim/utils.py (hunk 2 above)
Comment (k-rieck) [reply hisim/utils.py:56]: Thanks, I pushed again with an underscore

**Review by noah80 — APPROVED** [review]

**Pull request merged**